apps/encode_keychange.c

```diff
@@ -103,6 +103,15 @@ char           *snmp_getpassphrase(const char *prompt, int fvisible);
 static char    *getpass(const char *prompt);
 #endif
 
+static char *gethomedir() {
```

Comment:
Instead of duplicating the netsnmp_gethomedir() implementation, please call netsnmp_gethomedir(). apps link with libsnmp anyway.

Reply:
Done